doc/nrf/config_and_build/config_and_build_system.rst

```diff
@@ -101,7 +101,10 @@ The :file:`zephyr.dts` file contains the entire hardware-related configuration o
 The header file contains the same kind of information, but with defines usable by source code.
 
 For more information, see :ref:`configuring_devicetree` and Zephyr's :ref:`zephyr:dt-guide`.
-In particular, :ref:`zephyr:set-devicetree-overlays` explains how to use devicetree and its overlays to customize an application's devicetree.
+In particular, :ref:`zephyr:set-devicetree-overlays` explains how the base devicetree files are selected.
+
+In the |NCS|, you can :ref:`configure devicetree using the dedicated Devicetree Visual Editor <configuring_devicetree>`.
```

Comment:
Haven't read the docs on `Devicetree Visual Editor`, but I assume that it creates a file, and that file must either be given a name that is auto-picked during CMake time or else applied manual by the user.

Should we extend the sentence with such info ?

Reply:
@tejlmand  In VSCode The device tree editor will work on top of an overlay file the user has added or on the top most overlay file that the project uses. If this file is in the SDK we show a warning that you are editing an SDK file.

Does this answer you question?